test/app.e2e-spec.ts

```diff
@@ -124,6 +124,26 @@ describe('End-2-End Testing', () => {
         );
     });
 
+    it(
+        'invalid ticket by id',
+        async () => {
+            await request(app.getHttpServer())
+                .post('/api/v1/tickets')
+                .send(partyTicket)
+                .expect(201)
+                .then(async creationResponse => {
+                    const createdTicket = creationResponse.body;
+                    await request(app.getHttpServer())
+                        .delete('/api/v1/tickets/' + createdTicket.id)
+                        .send({
+                            ticketID: createdTicket.id,
+                        })
+                        .expect(200);
```

Comment:
```suggestion
                        .expect(204);
```
2xx mit 204 NoContent wäre besser.